app/uk/gov/hmrc/tai/connectors/TaxAccountConnector.scala

```diff
@@ -43,20 +42,28 @@ class TaxAccountConnector @Inject()(
   featureTogglesConfig: FeatureTogglesConfig)(implicit ec: ExecutionContext)
     extends HodsSource {
 
+  lazy val uuid = UUID.randomUUID().toString
```

Comment:
This will create the same uuid for every call, please have a review of val vs def

Reply:
Changed it to fucntion getUuid from val